src/app/(nextjs_migration)/components/client/FalseDoorBanner.tsx

```diff
@@ -82,26 +78,40 @@ type FalseDoorBanner = {
 };
 export const FalseDoorBanner = (props: FalseDoorBanner) => {
   const l10n = useL10n();
+  const [isLargeScreen, setIsLargeScreen] = useState(false);
+
+  const handleResize = () => {
+    // 1024px is equivalent to $screen-lg
+    if (window.innerWidth > 1024) {
+      setIsLargeScreen(true);
+    } else {
+      setIsLargeScreen(false);
+    }
+  };
```

Comment:
Q: We can't use media queries to control whether this is one line or two below (L101-ish)?